public/javascripts/checkout.js

```diff
@@ -0,0 +1,77 @@
+const stripe = Stripe('pk_test_rmALB2lGQQER0zyeD1O6wsAu');
```

Comment:
Requirement: Your Token should not be exposed in your repo. Please make this an Environmental Variable and regenerate a new token as this one has been compromised.

Reply:
@reactivepixel I need some help with this one

Reply:
see https://github.com/piazzaana/chicken-fries/issues/58